server/chat-commands/core.ts

```diff
@@ -544,11 +544,12 @@ export const commands: ChatCommands = {
 			return this.parse(`/pm ${targetUsername}, /invite ${targetRoom.roomid}`);
 		}
 
-		const targetUser = this.pmTarget!; // not room means it's a PM
+		const targetUser = this.pmTarget; // not room means it's a PM
 
 		if (!targetRoom) {
 			return this.errorReply(this.tr`The room "${target}" was not found.`);
 		}
+		if (!targetUser) return this.errorReply(this.tr`/invite - must be sent to a user or room`);
```

Comment:
This error seems unnecessarily vague to me? I would ``return this.parse(`/help invite`);``

Reply:
Okay.